packages/@webex/plugin-cc/test/unit/spec/services/core/aqm-reqs.ts

```diff
@@ -66,12 +65,22 @@ describe('AqmReqs', () => {
       webex: {} as any,
     }) as jest.Mocked<WebSocketManager>;
 
-    // Mock the addEventListener method
-    webSocketManagerInstance.addEventListener = jest.fn((event, callback) => {
-      if (event === 'message') {
-        webSocketManagerInstance.dispatchEvent = callback;
+    // Mock the on method to handle event listeners
+    const eventListeners: { [key: string]: Function[] } = {};
```

Comment:
Had to add this as we need to mock the EventEmitter framework somehow.
Open to suggestions on better ways, if there are.